Flaskerizer_src/gui_for_flaskerizer.py

```diff
@@ -0,0 +1,185 @@
+        filetypes=(("Html Files", "*.html"), ("all files", "*.*")))
+        self.html_location.set(path)
+        if self.validate_path(path):
+            self.html_location.set(path)
+        else:
+            self.html_location.set("Please select a valid html file")
+
+    def get_static_folder(self):
+        """Opens a file dialog prompting the user to select the static folder.
+        Gets the path, and sets the static_location to the path value.
+        """
+
+        path = filedialog.askdirectory(title="Select the main static folder")
+        if self.validate_path(path):
+            self.static_location.set(path)
+        else:
+            self.static_location.set("Please select a valid static folder")
+
+    def get_js_folder(self):
+        """Opens a file dialog prompting the user to select a JavaScript file.
+        Gets the directory path with the path_to_folder function
+        and sets the js_location variable with the returned path value.
+        """
+
+        path = filedialog.askopenfilename(title = "Select a .js file, normally inside the js folder",
+        filetypes=(("JavaScript Files", "*.js"), ("all files", "*.*")))
+        if self.validate_path(path):
+            self.js_location.set(path)
+        else:
+            self.js_location.set("Please select a valid js file")
+
+    def path_to_folder(self, path):
+        """Transforms a File path into a folder path
+        stripping the file name from the path,
+        leaving only the directory path.
+        """
+
+        return os.path.split(path)[0]
+    def validate_path(self, path):
+        """Validates the path making sure it exists
+        """
+        if os.path.exists(path):
+            return True
+        else:
+            return False
+
+    def validate_entries(self):
+        """Validates all entries and if
+        there is errors a error pop-up window appears
+        """
+        self.error_entries = []
+        html = self.validate_path(self.html_location.get())
+        static = self.validate_path(self.static_location.get())
+        js = self.validate_path(self.js_location.get())
+        if not html:
+            self.error_entries.append("'Html File location' ")
+        if not static:
+            self.error_entries.append("'Static Files location' ")
+        if not js:
+            self.error_entries.append("'JavaScript File location' ")
+        if not html or not static or not js:
+            self.error_message = "There is errors in the entries: "
+            for x in self.error_entries:
+                self.error_message = self.error_message + x 
+            messagebox.showerror("Entries Error", self.error_message)
+            return False
+        else:
+            return True
+
+    def get_values(self):
+        """Gets the html_location, static_location and js_location
+        values and sets them to new variables then uses them
```

Comment:
Gets the html_location, static_location and js_location
values for paths to the HTML files, 'static' folder content, and Javascript files of the Bootstrap template. These paths are passed to the StructureDirectory class for flaskerization.